process-services/latest/using/workflows.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Preconfigured workflows
+---
+
+Tasks and workflows help you keep track of the things you and other users need to do. You can create a standalone task or workflow, or you can attach a file to it.
+
+A workflow is a process that controls a specific task, such as a document review. Each task can be assigned to one or more people when a workflow is being created. Workflow creators can select to automatically send a notification email to users that are assigned a task as part of the workflow, but whether an email is sent or not the task will be visible in the users My Tasks.
+
+Once all the task actions required by a workflow have been finished, the workflow will move from active to completed status. At this point you’re free to delete the workflow.
+Individual users manage their own tasks, and the person who created a workflow manages the workflow.
+
+## Creating a tenant
+
+You need to create a tenant before you can start creating workflows. 
+
+To create a tenant, complete the following steps.
+
+1. Log in as admin to Alfresco Process Services as the admin user.
+
+2. Select **Identity Management**.
+
+3. Select **Tenants**.
+
+4. Select **Create tenant**.
+
+      The **Create tenant** dialog box appears.
+
+5. Enter the following details as per your requirements.
+
+      * Enter the **Name** – The name of your tenant, for example, xxx.
+      * Specify the **Maximum allowed users in this tenant** - This is set at 10 bBy default, it is set at 10.
+
+6. Select the **Deploy preconfigured workflows** checkbox.
+
+7. Add the details about the tenant owner including **First name**, **Last name**, **Email**, and **Password**. 
+
+      You can also assign a tenant owner from the list of created users.
+
+8. Click **Save**.
+

```

Comment:
**## To Change the owner of PreConfigured_workflows**

first the owner of the PreConfigured workflow needs to manually delete the workflow app and then the owner can be changed by going into tenant section click on edit tenant and add a new user or select from the exisiting user now after saving this user will be the owner of the workflow in the current tenant